.hound.yml

```diff
@@ -0,0 +1,4 @@
+eslint:
```

Comment:
How do I confirm that this works?

Reply:
@erickyvand ☝️

Reply:
I saw in hound documentation that you have to add enable: true, and add .eslintrc so that hound can work with eslint.

Reply:
Ok. Can you resolve the conflict you have?

Reply:
Okay let me solve it